src/app/shared/components/notification-box/notification-box-item.component.html

```diff
@@ -1,26 +1,30 @@
-<mdl-list-item>
-  <mdl-list-item-primary-content>
+<md-list-item>
+  <h4 md-line class="mat-list-item-content">
```

Comment:
Please add some css to achieve the same behaviour for this block without using h4